### ISSUE TITLE
IMDB Sentimental Analysis

### ISSUE THREAD

**Issue by keshav340** [Jayshah6699/datascience-mashup#40]
I can work on this issue.@Jayshah6699  add the labels and then assign it to me.

**Comment by Jayshah6699** [comment]
explain the project first mention the dataset and give brief summary. @keshav340

**Comment by keshav340** [comment]
The IMDB  dataset has 50,000 movie review in which we have to predict whether this review is positive or negative. We download this dataset from Keras .dataset.We are doing it through Neural Networks.
In this project, we have to clean the data like convert words into numbers, vectorizing. Then we define our model architecture and train the model. then we predict the result.
